thumbservice.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 from flask import Flask, request, abort, jsonify, send_file
-from flask_cors import CORS
+from flask.ext.cors import CORS
```

Comment:
This results in deprecation warnings. Make sure your flask installation is up to date.